### ISSUE TITLE
Insufficient contrast detected by Pa11y.

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/o-ft-affiliate-ribbon#13]
```
✖ Executing Pa11y
    →
    →  • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text
 in this element has a contrast ratio of 1.19:1. Recommendation: change background to #767473.
    →    ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    →    ├── html > body > div > div > div > span
    →    └── <span class="o-ft-affiliate-ribbon__text">Published by</span>
    → Failed Pa11y tests
```

The color is defined at https://github.com/Financial-Times/o-ft-affiliate-ribbon/blob/master/src/scss/_colors.scss#L1

**Comment by JakeChampion** [comment]
Fixed via #12
